src/charm.py

```diff
@@ -50,6 +55,11 @@ def __init__(self, *args):
         self._container = self.unit.get_container(self._container_name)
         self._hydra_relation_name = "endpoint-info"
         self._kratos_relation_name = "kratos-endpoint-info"
+        self._prometheus_scrape_relation_name = "metrics-endpoint"
+        self._loki_push_api_relation_name = "logging"
+        self._login_ui_service_command = "/usr/bin/identity-platform-login-ui"
+        self._log_dir = "/var/log"
+        self._log_path = f"{self._log_dir}/ui.log"
 
```

Comment:
Hope you won't take me for a nitpicker. I try to encourage, and implement that we stay consistent with our approaches across our operators. We usually use path objects for dirs. and files. I suggest this:
```
        self._log_dir = Path("/var/log")
        self._log_path = self._log_dir / "ui.log"
```
on line 99 this is why you see the string conversion to _log_path

Reply:
absolutely fair, dropping commit

Reply:
right so i have to review this in light of the following

```
  File "/var/lib/juju/agents/unit-identity-platform-login-ui-operator-0/charm/venv/yaml/representer.py", line 58, in represent_data
    node = self.yaml_representers[None](self, data)
  File "/var/lib/juju/agents/unit-identity-platform-login-ui-operator-0/charm/venv/yaml/representer.py", line 231, in represent_undefined
    raise RepresenterError("cannot represent an object", data)
yaml.representer.RepresenterError: ('cannot represent an object', PosixPath('/var/log/ui.log'))
```

in the code we don't really use it if not when is a string so here's what i m gonna do

```
        self._log_dir = "/var/log"
        self._log_path = f"{self._log_dir}/ui.log"
        assert Path(self._log_path)
```

happy to chat over this @canonical/identity